src/include/fmt_mot.h

```diff
@@ -0,0 +1,21 @@
+#ifndef _FMT_MOT_H_
+#define _FMT_MOT_H_
+
+#include <sys/types.h>
+
+typedef unsigned short MOTION_ARCHIVE;
+typedef unsigned short MOTION_TABLE;
+
+typedef struct MOTION_SEGMENT
```

Comment:
This should share a file with MOTION_CONTROL ideally.

Reply:
That makes sense. The idea here is was to separate out data format definitions into more self-contained headers. I know that ``fmt_hzd.h`` & ``fmt_mot.h`` existed in the OG source thanks to the preprocessor memleak with the PCX structure.